.github/workflows/CI.yaml

```diff
@@ -25,7 +27,9 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 22
-      - uses: oven-sh/setup-bun@v1
+      - uses: oven-sh/setup-bun@v2
+        with:
+          bun-version: "1.2.19"
```

Comment:
Should we add a comment to https://github.com/cubing/icons?tab=readme-ov-file#build-the-project so people know which version of bun to use? Don't have to duplicate the version, but could just point people here.

Reply:
Ok